.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
```

Comment:
Would you mind splitting out the hook update into a separate PR? It's good practice and easier to figure out if things go wrong if this is not intermingled with feature additions.

Reply:
@Balandat I am not too familiar with the hook updates. Do you mean the changes in .pre-commit-config.yaml or changes to all the files by pre-commit (in commit [c18e6d8](https://github.com/cornellius-gp/gpytorch/pull/2235/commits/c18e6d8e13356c9f9b17bf5e03e3877938bb6c07))

Reply:
I'm not sure why we need to change the `pre-commit-hooks` file at all. Can we get rid of these changes?

Reply:
Yes, removed it.